### PR TITLE
Convert validators and Config to Pydantic 2

### LINKED ISSUE #60: Pydantic validators warnings
When you run tests there are some warnings relative to code that will be removed  when pydantic 3 will be released:

image

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- Fix deprecated Pydantic 1 code (closes #60 )

### How to test:
- Run tests locally
- Run `docker-compose up`
- Go to  http://localhost:8000/docs

### Expected outcome:
- When you run tests locally you shouldn't get any warnings
- Docker-compose up command should serve the app locally

### Review:
- [ ] Code approved by
- [x] Tests executed by CR

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
Tests on the branch don't trigger any warnings: 🆗 

image


Dockerfile is built and image is runned without errors: 🆗 

image

image

File: schug/models.py
Comment (henrikstranneheim) [inline schug/models.py:1]: Remove?

**Review by henrikstranneheim — APPROVED** [review]
👍

File: schug/models.py
Comment (northwestwitch) [reply schug/models.py:1]: It's an empty unused file, so yes?

**Pull request merged**